.travis.yml

```diff
@@ -1,4 +1,5 @@
-sudo: false
+sudo: required
+dist: trusty
 language: node_js
 node_js:
   - '5'
```

Comment:
Perhaps we should build on travis with node 5 and 6 since 5 will be retired within 6 weeks.